### PR TITLE
Use ptrs to `ir.ACLRule`

### PR DESCRIPTION
**Pull request by YairSlobodin1**

### CODE REVIEW

File: README.md
Comment (zivnevo) [inline README.md:1]: why is there a diff in this file?
please revert

File: pkg/ir/acl.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ type ACLWriter interface {
 	WriteACL(aclColl *ACLCollection, vpc string) error
 }
 
-func (r *ACLRule) isRedundant(rules []ACLRule) bool {
+func (r *ACLRule) isRedundant(rules []*ACLRule) bool {
 	for i := range rules {
 		if rules[i].mustSupersede(r) {
 			return true
```
Comment (zivnevo) [inline pkg/ir/acl.go:51]:
```suggestion
	for _, rule := range rules {
		if rule.mustSupersede(r) {
			return true
```

File: pkg/io/confio/acl.go [hunk 2]
```diff
@@ -102,7 +102,7 @@ func aclRules(acl *ir.ACL) ([]vpcv1.NetworkACLRuleItemIntf, error) {
 			Href: ref.Href,
 			ID:   ref.ID,
 		}
-		rule, err := makeACLRuleItem(&rules[i], current, next)
+		rule, err := makeACLRuleItem(rules[i], current, next)
```
Comment (zivnevo) [inline pkg/io/confio/acl.go:105]:
```suggestion
		rule, err := makeACLRuleItem(rule, current, next)
```
here too

**Review by zivnevo — CHANGES_REQUESTED** [review]

File: pkg/io/confio/acl.go (hunk 2 above)
Comment (YairSlobodin1) [reply pkg/io/confio/acl.go:105]: it is in reverse order (n, n-1, ...,1)

File: README.md
Comment (YairSlobodin1) [reply README.md:1]: I thought it would be clearer that each **Note** would start on a new line

Comment (zivnevo) [reply README.md:1]: ah, ok. I missed this changed. I just saw weird additional spaces all around.
Yes, having each note on a different line is better.

Comment (YairSlobodin1) [reply README.md:1]: done

**Review by zivnevo — APPROVED** [review]

**Pull request merged**